spec/factories/projects.rb

```diff
@@ -1,54 +1,65 @@
+def yaml_load(file)
+  YAML.load(File.read(File.expand_path(file)))
+end
 FactoryGirl.define do
-  factory :project, :class => Tmuxinator::Project do
+  factory :project, class: Tmuxinator::Project do
     transient do
-      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/sample.yml")}")) }
+      file { yaml_load('spec/fixtures/sample.yml') }
     end
 
     initialize_with { Tmuxinator::Project.new(file) }
   end
 
-  factory :project_with_force_attach, :class => Tmuxinator::Project do
+  factory :project_with_force_attach, class: Tmuxinator::Project do
     transient do
-      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
+      file { yaml_load('spec/fixtures/detach.yml') }
     end
 
-    initialize_with { Tmuxinator::Project.new(file, {:force_attach => true}) }
+    initialize_with { Tmuxinator::Project.new(file, force_attach: true) }
   end
 
-  factory :project_with_force_detach, :class => Tmuxinator::Project do
+  factory :project_with_force_detach, class: Tmuxinator::Project do
     transient do
-      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
+      file { yaml_load('spec/fixtures/detach.yml') }
     end
-    initialize_with { Tmuxinator::Project.new(file, {:force_detach => true}) }
+    initialize_with { Tmuxinator::Project.new(file, force_detach: true) }
   end
 
-  factory :project_with_custom_name, :class => Tmuxinator::Project do
+  factory :project_with_custom_name, class: Tmuxinator::Project do
     transient do
-      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/sample.yml")}")) }
+      file { yaml_load('spec/fixtures/sample.yml') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.